pkg/util/mount/mount_windows.go

```diff
@@ -383,13 +383,13 @@ func getAllParentLinks(path string) ([]string, error) {
 func (mounter *Mounter) GetMountRefs(pathname string) ([]string, error) {
 	windowsPath := normalizeWindowsPath(pathname)
 	pathExists, pathErr := PathExists(windowsPath)
-	if !pathExists {
-		return []string{}, nil
-	} else if IsCorruptedMnt(pathErr) {
+	if IsCorruptedMnt(pathErr) {
```

Comment:
this code is fine, but I am thinking to simplify it like
if !pathExists {
    //might also log err if err != nil if needed
     return []string{}, err
}
return []string{pathname}, nil

Reply:
The same concern from `mount_linux` is also valid for here.